cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/batch/source/DBSource.java

```diff
@@ -77,14 +79,16 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   }
 
   @Override
-  public void prepareRun(BatchSourceContext context) {
+  public void prepareRun(BatchSourceContext context) throws Exception {
     LOG.debug("pluginType = {}; pluginName = {}; connectionString = {}; importQuery = {}; " +
                 "countQuery = {}",
               dbSourceConfig.jdbcPluginType, dbSourceConfig.jdbcPluginName,
               dbSourceConfig.connectionString, dbSourceConfig.importQuery, dbSourceConfig.countQuery);
 
-    Job job = context.getHadoopJob();
+    Job job = Job.getInstance();
     Configuration hConf = job.getConfiguration();
+    hConf.clear();
```

Comment:
why is this needed? So the ``hConf`` passed to ``SourceInputFormatProvider`` on line 101 does not have anything other than ``DBConfiguration``, ``ETLDBInputFormat`` properties?